brainscore/benchmarks/__init__.py

```diff
@@ -193,6 +193,8 @@ def _experimental_benchmark_pool():
     pool['dicarlo.SanghaviMurty2020.IT-pls'] = LazyLoad(DicarloSanghaviMurty2020ITPLS)
     from .rajalingham2020 import DicarloRajalingham2020ITPLS
     pool['dicarlo.Rajalingham2020.IT-pls'] = LazyLoad(DicarloRajalingham2020ITPLS)
+    from .lee2020 import DicarloLee2020ITSpatialCorrelation
+    pool['dicarlo.Lee2020.IT-spatial_correlation'] = LazyLoad(DicarloLee2020ITSpatialCorrelation)
```

Comment:
I think this should be `dicarlo.MajajHong2015.IT-spatial_correlation` because that is where the data came from. Ideally the characterization in Lee et al. 2020 as well as you as the builder of comparison metric should be featured somehow as well; for now that should go into the Metric description I think.

This is a discussion we should also have with Jim. My preference is to credit: 1) data provider (from the paper that the data was uploaded to brain-score from), 2) metric builder (paper where the metric for brain-model comparison was introduced), 3) benchmark implementer

Reply:
do you want me to move the whole benchmark code to brainscore/benchmarks/majajhong2015 then?